src/Components/ForgotPassword.js

```diff
@@ -26,7 +14,7 @@ function ForgotPassword() {
         type="email"
         name="email"
       ></input>
-      <button onClick={passwordReset}>Send</button>
+      <button onClick={passwordReset({ email }, history)}>Send</button>
```

Comment:
Wrong implementtaion onClick requires a function not a function call , use ()=> passwordReset({email}, history)